build.sbt

```diff
@@ -6,13 +6,14 @@ scalaVersion := "2.11.7"
 
 libraryDependencies += "org.spire-math" %% "cats" % "0.2.0"
 libraryDependencies += "com.lihaoyi" %% "pprint" % "0.3.6"
+libraryDependencies += "com.lihaoyi" %% "upickle" % "0.3.6"
 libraryDependencies += "com.chuusai" %% "shapeless" % "2.2.5"
 
 resolvers += Resolver.sonatypeRepo("releases")
 
 addCompilerPlugin("org.spire-math" %% "kind-projector" % "0.6.3")
 
-wartremoverErrors ++= Warts.allBut(Wart.Any, Wart.Nothing, Wart.Throw, Wart.IsInstanceOf, Wart.AsInstanceOf, Wart.NoNeedForMonad)
+// wartremoverErrors ++= Warts.allBut(Wart.Any, Wart.Nothing, Wart.Throw, Wart.IsInstanceOf, Wart.AsInstanceOf, Wart.NoNeedForMonad)
```

Comment:
Yep, that should be already possible :) 

I wonder if the wart remover was a good idea to have.. It works very well for the code in this repo but also limits the libraries we can use and a couldn't find a good way to disable it for implicit lookups from ext libs...